internal/kafka/internal/config/dynamic_scaling_config.go

```diff
@@ -5,44 +5,58 @@ import (
 )
 
 type DynamicScalingConfig struct {
-	filePath      string
-	Configuration map[string]InstanceTypeDynamicScalingConfig
+	filePath                                      string
+	ComputeNodesPerInstanceType                   map[string]InstanceTypeDynamicScalingConfig `yaml:"compute_nodes_per_instance_type"`
+	MachineTypePerCloudProvider                   map[string]MachineTypeConfig                `yaml:"machine_type_per_cloud_provider"`
```

Comment:
Can we use `cloudproviders.CloudProviderID` type as the key?

Reply:
Good idea. Let me see if how that one goes.

Reply:
done